src/modules/swap/ModuleSwap.vue

```diff
@@ -47,7 +47,13 @@
             /></v-col>
 
             <v-col cols="12" sm="2" class="pt-0 pt-sm-3">
-              <div class="d-flex align-center justify-center">
+              <div
+                :class="[
```

Comment:
add disabled computed property on it:

disabled if reverse pair does not exists